composer.json

```diff
@@ -25,10 +25,19 @@
                 }
             }
         },
-	{
-	    "type": "vcs",
-	    "url": "https://github.com/digitalutsc/advanced_search.git"
-	}
+	    {
+            "type": "package",
+            "package": {
+              "name": "islandora/advanced_search",
+              "version": "2.0.0-beta3",
```

Comment:
Why specify this commit? The starter site should update with new versions (non-breaking).